src/components/Picker/Picker.tsx

```diff
@@ -246,6 +195,26 @@ class Picker extends React.Component<Props, State> {
         });
       },
 
+      storeInputReference: (autosuggest: Autosuggest) => {
+        if (autosuggest !== null) {
+          if (this.state.input !== autosuggest.props.inputProps.value) {
+            this.setState({ input: autosuggest.props.inputProps.value });
+          }
+        }
+      },
+
+      storeFocus: (e: HTMLElement) => {
+        if (!this.state.focusArr.includes(e) && e !== null) {
+          const focusArr = this.state.focusArr.length ? this.state.focusArr.concat([e]) : [e];
+          this.setState({ focusArr });
+        }
+      },
+
+      onClick: (e: React.FormEvent<Element>) => {
+        // TODO can write a code accoring to requirements
```

Comment:
Picker click event remove it as discussed and make sure it works

Reply:
resolved